packages/shared/styles/components/SfChevron.scss

```diff
@@ -50,4 +44,19 @@ $chevron__bar-color: $c-black !default;
   cursor: pointer;
   @include chevron-bar($half-name: left);
   @include chevron-bar($half-name: right);
+
+  &--top {
```

Comment:
@gibkigonzo can you update `@mixin chevron-bar` read about `@at-root` and you can use `$this:&` to bind `.sf-chevron `

Reply:
Cool stuff :) Thanks for advice